finding/finding.go

```diff
@@ -266,6 +266,16 @@ func (f *Finding) WithRemediationMetadata(values map[string]string) *Finding {
 	return f
 }
 
+// WithValue adds a value to f.Values.
+// No copy is made.
+func (f *Finding) WithValue(k string, v int) *Finding {
```

Comment:
I know the existing code use values of type in https://github.com/ossf/scorecard/blob/main/finding/finding.go#L226, but do we expect it may require another types in certain cases? strings?

(not a blocker for this PR)

Reply:
ATM, I don't see a case for other types.

Reply:
Closing since there doesn't seem to be an issue. Feel free to re-open if that is not the case.